.github/workflows/tests.yml

```diff
@@ -248,7 +248,7 @@ jobs:
           coverage run -p -m unittest discover -s src
           PURE_PYTHON=1 coverage run -p -m unittest discover -s src
           coverage combine
-          coverage report -i
+          coverage report --ignore-errors --show-missing
```

Comment:
:100:  (I had to look up `-i` before)

Actually I was so busy looking up what -i meant that I forgot to ask why we needed it here?

Reply:
I thought previously there was a comment in the file to explain it. But I was not able to find it. Maybe in another repository.